### ISSUE TITLE
Create shapefile of Desert Spring Locations

### ISSUE THREAD

**Issue by kklausmeyer** [tnc-ca-geo/gd_intern#93]
Connor-

I have enclosed a PDF with a long table of spring locations (Table 1).  I contacted the author to get an excel file of the Table 1 - spring locations, but unfortunately, the author of the report has not been responsive for months.  So I would like you to convert the PDF file to an excel file, and then ultimately a point feature class.  

I tested it, and this method of converting the table seems to work:  https://www.economicsnetwork.ac.uk/tips/pdf2excel

Basically copy the table from PDF to Word, and then Word to Excel.  

Then use the Lat and Long data to make a point feature class and save the feature class here:
N:\Projects\Groundwater\shapes.

Please reproject the final data to California Albers projection, and compare to the maps in the PDF to make sure it looks right.  

Deadline is flexible but I would like it by September 22nd if possible.  The board materials is higher priority than this.

**Comment by kklausmeyer** [comment]
[AZI 2016 Mojave Spring Survey Report.pdf](https://github.com/tnc-ca-geo/gd_intern/files/1282825/AZI.2016.Mojave.Spring.Survey.Report.pdf)

**Comment by connorshank** [comment]
Table and shapefile have been created and placed in:
N:\Projects\Groundwater\shapes\mojave_springs_survey.shp
N:\Projects\Groundwater\tables\Mojave_Spring_Survey.csv

**Comment by kklausmeyer** [comment]
Data looks good but projection system seems off.  Connor to reimport and test against a basemap.

**Comment by connorshank** [comment]
Re-projected and QC checked the data and resulting layer, same name and location as above, replaced the old shapefile.

**Comment by kklausmeyer** [comment]
Looks great Connor-  two more steps:

1. Now that you have the correct projection assigned, use the "project" tool to reproject to California Albers
2. Write some simple metadata in the shapefile using the ArcGIS format- give the source of the data, what you did, and provide my name as a contact person

**Comment by connorshank** [comment]
Done, the Alber's projected file is located at:
N:\Projects\Groundwater\shapes\mojave_springs_survey_Albers.shp
